projection.go

```diff
@@ -61,18 +63,18 @@ type ProjectionConfigurer interface {
 	// Each handler within an application MUST have a unique, non-empty name.
 	Name(n string)
 
-	// AcceptsEventType configures the engine to route event messages of the
+	// ConsumesEventType configures the engine to route event messages of the
 	// same type as m to the handler.
 	//
 	// It MUST be called at least once within a call to Configure(). It MUST NOT
 	// be called more than once with an event message of the same type.
 	//
-	// Multiple handlers within a single application MAY receive event messages
-	// of the same type.
+	// Multiple handlers within an application MAY consume event messages of the
+	// same type.
```

Comment:
So far all the changes make sense to me. I just wonder if the statement above will end up in process iface descriptions. I assume the "produce" and "consume" terminology terminology did not make it to aggregates and processes yet.

Reply:
I've done aggregates now, but not processes yet.